cmake/AddInputFileTests.cmake

```diff
@@ -33,10 +35,23 @@ function(add_single_input_file_test INPUT_FILE EXECUTABLE CHECK_TYPE TIMEOUT)
       --check-options --input-file ${INPUT_FILE}
       )
   elseif("${CHECK_TYPE}" STREQUAL "execute")
+    # Write command into shell script to make it easier to chain multiple
+    # commands
+    file(
```

Comment:
Please don't write one file per test, just write one file and then pass the variable things into it as arguments (the executable and input file).